sources/packages/backend/apps/workers/src/controllers/assessment/assessment.dto.ts

```diff
@@ -139,3 +140,8 @@ export interface UpdateNOAStatusHeaderDTO {
 export interface UpdateNOAStatusHeaderDTO {
   status: AssessmentStatus;
 }
+
+export interface SaveWorkflowDataJobInDTO {
+  [ASSESSMENT_ID]: number;
+  workflowData: WorkflowData;
```

Comment:
The variable should be declared with the const `WORKFLOW_DATA` following the same from `ASSESSMENT_ID`.